### PR TITLE
Add a property for default sidecar mem limit

### PR DESCRIPTION
**Pull request by garagatyi**
### What does this PR do?
Add a property for default sidecar mem limit

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: deploy/kubernetes/helm/che/templates/configmap.yaml [hunk 1]
```diff
@@ -80,6 +80,9 @@ data:
 {{- if .Values.workspaceDefaultRamRequest }}
   CHE_WORKSPACE_DEFAULT_MEMORY_REQUEST_MB: {{ .Values.workspaceDefaultRamRequest }}
 {{- end }}
-{{- if .Values.workspaceDefaultRamRequest }}
-  CHE_WORKSPACE_DEFAULT_MEMORY_LIMIT_MB: {{ .Values.workspaceDefaultRamRequest }}
+{{- if .Values.workspaceDefaultRamLimit }}
+  CHE_WORKSPACE_DEFAULT_MEMORY_LIMIT_MB: {{ .Values.workspaceDefaultRamLimit }}
+{{- end }}
+{{- if .Values.workspaceSidecarDefaultRamLimit }}
+  CHE_WORKSPACE_DEFAULT_MEMORY_LIMIT_MB: {{ .Values.workspaceDefaultRamLimit }}
```
Comment (sleshchenko) [inline deploy/kubernetes/helm/che/templates/configmap.yaml]: I think it's a copy/paste error and env name and value should be changed to sidecar related

File: deploy/kubernetes/helm/che/templates/deployment.yaml [hunk 2]
```diff
@@ -283,6 +283,13 @@ spec:
             configMapKeyRef:
               key: CHE_WORKSPACE_DEFAULT_MEMORY_LIMIT_MB
               name: che
+{{- end }}
+{{- if .Values.workspaceSidecarDefaultRamLimit }}
+        - name: CHE_WORKSPACE_DEFAULT_MEMORY_LIMIT_MB
```
Comment (sleshchenko) [inline deploy/kubernetes/helm/che/templates/deployment.yaml]: Copy/paste error?

File: deploy/kubernetes/helm/che/templates/configmap.yaml (hunk 1 above)
Comment (garagatyi) [reply deploy/kubernetes/helm/che/templates/configmap.yaml]: ur right

File: deploy/kubernetes/helm/che/templates/deployment.yaml (hunk 2 above)
Comment (garagatyi) [reply deploy/kubernetes/helm/che/templates/deployment.yaml]: exactly

**Review by l0rd — APPROVED** [review]

**Review by skabashnyuk — APPROVED** [review]

**Comment by garagatyi** [comment]
@sleshchenko I've fixed lines you had commented

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**